core/model/configs.go

```diff
@@ -50,6 +53,11 @@ type EnvConfigData struct {
 	AnalyticsToken string `json:"analytics_token" bson:"analytics_token"`
 }
 
+// AuthConfigData contains auth configs for this service
+type AuthConfigData struct {
+	ExternalID string `json:"external_id" bson:"external_id"`
```

Comment:
I think we can probably just put this in the env config rather than introducing a new config doc.